### DISCUSSION TITLE
Where are the Img2Img + Inpaint codes?

### DISCUSSION THREAD

**Discussion by kadirnar** [Mikubill/sd-webui-controlnet#1219] [General]
Can I do this using the Diffusers library? For example, I want to use the inpaint feature for the openjourney model.

**Comment by lllyasviel** [comment]
hi the inpaint code directly use a1111 inpaint https://github.com/AUTOMATIC1111/stable-diffusion-webui/blob/5ab7f213bec2f816f9c5644becb32eb72c8ffb89/modules/processing.py#L1101

**Reply by kadirnar** [reply]
I am using the inpaint upload feature. I debug the code. self.inpainting_fill=1 so it doesn't go into the if loop.

**Comment by lllyasviel** [comment]
https://github.com/lllyasviel/ControlNet-v1-1-nightly/issues/57

**Reply by kadirnar** [reply]
This is great development. Can you add this to the [controlnet ](https://github.com/lllyasviel/ControlNet)library?

**Reply by kadirnar** [reply]
Does it support Diffusion Model + Controlnet + Depth + Inpaint? https://github.com/Mikubill/sd-webui-controlnet/discussions/1143#discussioncomment-5811501 https://github.com/Mikubill/sd-webui-controlnet/discussions/1143#discussioncomment-5788624

**Reply by lllyasviel** [reply]
hi it is from your linked repo i think
